snmp/datadog_checks/snmp/data/default_profiles/chatsworth_pdu.yaml

```diff
@@ -497,6 +503,40 @@ metrics:
         column:
           OID: 1.3.6.1.4.1.30932.1.10.1.7.100.1.2
           name: cpiPduLockId
+  - MIB: CPI-UNITY-MIB
+    table:
+      OID: 1.3.6.1.4.1.30932.1.10.1.7.100
+      name: cpiPduOutletTable
+    symbols:
+      - name: cpiPduEas
+        constant_value_one: true
+    metric_tags:
+      - tag: lock_id
+        column:
+          OID: 1.3.6.1.4.1.30932.1.10.1.7.100.1.2
+          name: cpiPduLockId
+      - tag: eas_status
+        column:
+          OID: 1.3.6.1.4.1.30932.1.10.1.7.100.1.3
+          name: cpiPduEasStatus
+        mapping:
+          0: inactive
+          1: ready
+          2: error
+      - tag: door_status
+        column:
+          OID: 1.3.6.1.4.1.30932.1.10.1.7.100.1.4
+          name: cpiPduDoorStatus
+        mapping:
+          0: closed
+          1: open
+      - tag: lock_status
+        column:
+          OID: 1.3.6.1.4.1.30932.1.10.1.7.100.1.5
+          name: cpiPduLockStatus
+        mapping:
+          0: closed
+          1: open
```

Comment:
@AlexandreYang I think now the situation is the same as [here](https://github.com/DataDog/integrations-core/pull/15491#discussion_r1285737880) ?

Reply:
I'm not sure to fully understand :) Could you clarify a bit?

Reply:
Sorry, it's a different case, just realized that 😅